File: Backend-POS/POS.Main/POS.Main.Business.HumanResource/Models/EmployeeAddress/EmployeeAddressResponseModel.cs
using POS.Main.Core.Enums;

namespace POS.Main.Business.HumanResource.Models;

public class EmployeeAddressResponseModel
{
    public int AddressId { get; set; }
    public int EmployeeId { get; set; }
    public EAddressType AddressType { get; set; }
    public string AddressTypeName => AddressType.ToString();
    public string? HouseNumber { get; set; }
    public string? Building { get; set; }
    public string? Moo { get; set; }
    public string? Soi { get; set; }
    public string? Yaek { get; set; }
    public string? Road { get; set; }
    public string? SubDistrict { get; set; }
    public string? District { get; set; }
    public string? Province { get; set; }
    public string? PostalCode { get; set; }
}

File: Frontend-POS/RBMS-POS-Mobile-Web/src/app/core/api/models/employee-address-response-model.ts
/* tslint:disable */
/* eslint-disable */
import { EAddressType } from '../models/e-address-type';
export interface EmployeeAddressResponseModel {
  addressId?: number;
  addressType?: EAddressType;
  addressTypeName?: string | null;
  building?: string | null;
  district?: string | null;
  employeeId?: number;
  houseNumber?: string | null;
  moo?: string | null;
  postalCode?: string | null;
  province?: string | null;
  road?: string | null;
  soi?: string | null;
  subDistrict?: string | null;
  yaek?: string | null;
}
